site/assets/scss/_toc.scss

```diff
@@ -10,6 +10,10 @@
     overflow-y: auto;
```

Comment:
I don't see the cutoff on the left, only on the right. Given that, wondering if it's simpler to just add `padding-right: .125rem;` around here? Curious why I'm seeing a different cropping than you though.

Reply:
different window sizes? let me get a quick screen recording of what i'm seeing...